pkg/resources/tag_association.go

```diff
@@ -169,12 +152,12 @@ func CreateContextTagAssociation(ctx context.Context, d *schema.ResourceData, me
 		if !skipValidate {
 			log.Println("[DEBUG] validating tag creation")
 			if err := retry.RetryContext(ctx, d.Timeout(schema.TimeoutCreate)-time.Minute, func() *retry.RetryError {
```

Comment:
is this function from some kind of library? If yes, I think we should use our own retry to slowly reduce our usage of external libraries.

Reply:
It's a helper function from the plugin sdk.